src/views/Verification.vue

```diff
@@ -0,0 +1,172 @@
+<template>
+  <div
+    class="
+      min-h-screen
+      bg-[url('/images/beach-product-background.png')] bg-bottom bg-cover
+      text-gray-800
+      flex flex-col
+      justify-center
+      relative
+      overflow-hidden
+      sm:py-12
+    "
+  >
+    <div class="relative sm:max-w-xl mx-auto text-center">
+      <div
+        class="mt-4 bg-rgba(255, 255, 255, 0.3) shadow-2xl rounded-xl text-left"
+      >
```

Comment:
Refactor repeated codes like these to the `Auth.vue` file, that's the whole point of nested routes.